### PR TITLE
[enhance](S3) Init default retry strategy for aws s3 sdk

### PR DESCRIPTION
**Pull request by ByteYue**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by dataroaring — APPROVED** [review]
LGTM

File: be/src/common/config.cpp [hunk 1]
```diff
@@ -1176,6 +1176,8 @@ DEFINE_Int32(partition_disk_index_lru_size, "10000");
 
 DEFINE_mBool(check_segment_when_build_rowset_meta, "false");
 
+DEFINE_mInt32(max_s3_client_retry_times, "10");
```
Comment (gavinchou) [inline be/src/common/config.cpp:1179]: "_times" seems redundant

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Comment by ByteYue** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**